Cargo.toml

```diff
@@ -54,7 +54,6 @@ atty = { version = "0.2.14", optional = true }
 cargo_metadata = "0.9.1"
 dirs = "2.0.2"
 env_proxy = "0.4.0"
-error-chain = "0.12.2"
 failure = "0.1.7"
```

Comment:
is it only used for backtraces? I find it strange to have 2 error libs. I would consider removing it.

Reply:
AFAIK, backtraces are still nightly-only in the stdlib, so if we want stable backtraces we'll have to stick with `failure` for now

Reply:
Also, the `cargo-metadata` crate uses failure and in order to use it's error type, we need the failure crate's compatibility conversion. So even if stop using failure for backtraces , we'll still need `failure` for that compatibility layer

Reply:
Update: I have a commit that removes `failure` completely. It adds a feature to `cargo-edit` that, when enabled, includes the nightly-only stdlib feature `backtrace`. If this feature is not enabled, then backtraces will not be printed.

I haven't pushed it up because I figured I'd get your thoughts first.

Reply:
Nah, I think we want cargo-edit to be stable-compilable. I only asked to see if it's relatively easy to remove it, if it's not, let's keep it.